.circleci/scripts/auto-rebase-branches.txt

```diff
@@ -1,3 +1,2 @@
-jz/rebase-custom1
-jz/rebase-custom2
-jz/rebase-custom3
+yogibo
+nmwa
```

Comment:
@Jiyuan-Bolt This is a dangerous strategy. These are (might be) the branches that are deployed to merchants' production environments. They are tested and stable and **not expected to change**. So the merchants may be used to updating the rest of the system through composer, not affecting Bolt functionality because their branches were "static". With this change they would update Bolt plugin as well, unwillingly and unknowingly perhaps, to a version that's (at least) not tested against their system. 
cc @usmanismail-bolt @matt-thomason @njfaries @vitaliyreznikov  @mccottry @johnmccombs1 @dumega

Reply:
This branches is something like master, right? We don't expect that merchants will install this. If I understand correctly when we create a new version `2.10.0` we should create `2.10.0-yogibo` and `2.10.0-nmwa`.
What if we call branches `yogibo-dev` and `nmwa-dev`?

Reply:
@danac-gs  Good point. 

@vitaliyreznikov  @Jiyuan-Bolt, @matt-thomason   We don't really need to push rebases to remote **yet**. We can just report if the rebase is successful or not how about if we don't push rebase to remote?

When we have [MX-1317](https://boltpay.atlassian.net/browse/MX-1317) completed we will have tests run as part of the circle ci job as well. That information is useful and then (for now) we can manually rebase.

Thoughts?

Reply:
Sounds good for me.

Reply:
@usmanismail-bolt and I discussed this over slack, and we think that auto-rebasing is still more useful than the above suggestion. We propose this:
- Move all merchants with custom branches to a standardized naming scheme, like `merchant/<some-branch-name>`
- Autorebase on a variant of that branch name per merchant, say like `ci/<some-branch-name`
- Periodically push good `ci` branches to `merchant` branches.